en_US/data-integration/syskeeper.md

```diff
@@ -0,0 +1,171 @@
+
+<img src="./assets/syskeeper_bridge_architecture.png" alt="syskeeper_bridge_architecture" style="zoom:67%;" />
+
+The workflow of the data integration is as follows:
+
+1. **Message publication and reception**
+
+   Devices, whether they are part of connected vehicles, IIoT systems, or energy management platforms, establish successful connections to EMQX through the MQTT protocol and send messages via MQTT based on their operational states, readings, or triggered events. When EMQX receives these messages, it initiates the matching process within its rules engine.
+2. **Message data processing**
+
+   When a message arrives, it passes through the rule engine and is then processed by the rule defined in EMQX. The rules, based on predefined criteria, determine which messages need to be routed to SysKeeper. If any rules specify payload transformations, those transformations are applied, such as converting data formats, filtering out specific information, or enriching the payload with additional context.
+3. **Data buffering**
+
+   EMQX provides an in-memory message buffer to prevent data loss when the Security Zone III is unavailable. Data is temporarily held in the buffer, and may be offloaded to disk to prevent memory overload. Note that data is not preserved if the data integration or the EMQX node is restarted.
+4. **Data ingestion to SysKeeper**
+
+   Once the rule engine identifies a message for SysKeeper, it triggers an action of forwarding the messages to zone III.
+5. **Data Utilization**
+
+   In zone III, the MQTT messages will be republish as it origin form, businesses can harness its querying power for various use cases.
+
+## Features and Benefits
+
+The data integration with SysKeeper offers a range of features and benefits tailored to ensure effective data handling and storage:
+
+- **Real-time Data Streaming**
+
+  EMQX is built for handling real-time data streams, ensuring efficient and reliable data transmission from source systems to SysKeeper. It enables organizations to capture and analyze data in real-time, making it ideal for use cases requiring immediate insights and actions.
+
+- **Flexible Data Transformation**
+
+  EMQX provides a powerful SQL-based Rule Engine, allowing organizations to pre-process data before passthrough it. It supports various data transformation mechanisms, such as filtering, routing, aggregation, and enrichment, enabling organizations to shape the data according to their needs.
+
+- **Scalability and High Throughput**
+
+  EMQX is architected for horizontal scalability, effortlessly managing the surging message traffic generated by an ever-expanding fleet of IoT devices. This solution effortlessly adapts to expanding data volumes and supports high-concurrency access. As a result, IoT time-series workloads can effortlessly manage the increasing requirements of data ingestion, storage, and processing as IoT deployments scale to unprecedented levels.
+
+## Before You Start
```

Comment:
1. Rename data bridge to sink
2. Some operations are aligned to the Dashboard
3.Update dir.yml

See: https://github.com/emqx/emqx-docs/blob/release-5.4/en_US/data-integration/data-bridge-mysql.md#:~:text=Select%20MySQL%20from%20the%20Type%20of%20Action%20dropdown%20list